lib/matplotlib/axes/_base.py

```diff
@@ -2056,7 +2042,7 @@ def draw(self, renderer=None, inframe=False):
         # if the minimum zorder is negative, start rasterization
         rasterization_zorder = self._rasterization_zorder
         if (rasterization_zorder is not None and
-            len(dsu) > 0 and dsu[0][0] < rasterization_zorder):
+                len(dsu) > 0 and dsu[0][0] < rasterization_zorder):
```

Comment:
That was me just going through pep8. I believe it may have been disabled already on travis side.